articles/google-play-in-app-reviews/index.md

```diff
@@ -0,0 +1,250 @@
+
+Another case would be that you have, say, an e-commerce application. In this case, it would be inappropriate to tell a user to review the app in the middle of making an order. You want the user to have enough information about your application services before submitting a review. In this case, you would ask a user to write a review when the order has been processed, to capture the right moment. You don’t want to show the review dialog when a user is yet to interact with your application enough to give feedback.
+
+With Google Core, the API is designed to respect user privacy. The rating sheet does not show up at any time. It is not attached to any call to action event. To limit API misuse, the API has quota limits per user. However, Google has not provided the exact number of times the rating dialog will open up per user. A similar iOS API has its quota limits set three times a year per user, which makes sense, as this will not be annoying to the user.
+
+Check the documentation [guidelines](https://developer.android.com/guide/playcore/in-app-review#when-to-request) regarding when you should request the review flow. The API states that you need to take care to note the following.
+
+1. How you ask for reviews matters. When using this API, you should not ask predictive or opinion questions before or after the review sheet dialogs show the app in your application. These questions include:
+
+    -  Would you rate this application five stars?
+    -  Do you like this app?
+
+2. This API also advises you only to show dialog once a user has enough experience with your application. It's not a good practice to ask a user to write a review before the app delivers value to that user. Chances are, the user will get annoyed and skip the review. And if they don't skip it, they will give negative reviews.
+
+3. In order to minimize API usage and avoid annoying users, it would be best if you did not excessively ask users to write reviews.
+
+### Implementation
+Google describes this API as light, and indeed it is. It is simple to integrate within your application. To begin, identify an activity where you want to implement the review flow. Once implemented, the API will handle the review flow for you.
+
+#### Requirements
+- An Android phone running android 5.0 (API 21) or higher. The device should have Google Play installed and connected with a Gmail account.
+- Google Play Core Library, version 1.8.0 or higher.
+
+#### Integration
+
+##### Step 1: Adding dependency
+Add Google Core Library to your `app.gradle` file.
```

Comment:
Please provide the Gradle dependency and specify in which gradle file to add the specific dependency. There are different levels of `build.gradle` files